CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Changelog
 
-## [Unreleased]
+# v0.2.0
 
 ## Added
 
 - Added emulation support for an i8042 controller that only handles the CPU
   reset.
+- Added `SerialEvents` trait, which can be implemented by a backend that wants
+  to keep track of serial events using metrics, logs etc.
+- Added a threat model to the serial console documentation.
+- Added emulation support for an ARM PL031 Real Time Clock, and the `RTCEvents`
+  trait, used for keeping track of RTC events.
+- Added an implementation for `Arc<EV>` for both serial console and RTC device.
+- Added methods for retrieving a reference to the events object for both serial
+  console and RTC device.
```

Comment:
Can you please add links to issue (or PRs when the issues are not available) so we can easily trace back the changes?